app/src/main/res/xml/pref_advanced_settings.xml

```diff
@@ -70,6 +70,12 @@
                 android:summary="@string/pref_summary_wear_logs_prefix"
                 android:textIsSelectable="true"
                 android:title="@string/pref_wear_logs_prefix" />
+            <CheckBoxPreference
```

Comment:
After this merge I'm planning to fold Pebble and Wear settings in to their own screens as the preference page is really busy with options now.